### ISSUE TITLE
Unclear that I had to generate a tasks.json and fill in launch.json

### ISSUE THREAD

**Issue by wugeorgeq** [dotnet/docs#9490]
I ran into errors debugging because I needed to generate a tasks.json file, also i needed to fill out launch.json with the path to app.dll

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3343b35c-90c1-2e11-498d-ee1d4eb409f6
* Version Independent ID: f2eda261-d4c3-c90a-1294-dfe11e55c341
* Content: [Getting started with .NET Core on macOS](https://docs.microsoft.com/en-us/dotnet/core/tutorials/using-on-macos)
* Content Source: [docs/core/tutorials/using-on-macos.md](https://github.com/dotnet/docs/blob/master/docs/core/tutorials/using-on-macos.md)
* Product: **dotnet-core**
* GitHub Login: @bleroy
* Microsoft Alias: **mairaw**

**Comment by mairaw** [comment]
Thanks for your feedback @wugeorgeq. Which version of .NET Core you're using?
VS Code should create those for you when running the commands and launching the debugger.

**Comment by mairaw** [comment]
Also, can you run this one successfully?
https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio-code

**Comment by SurlyMae** [comment]
I'm having this same issue.
`.NET Core SDK (reflecting any global.json):
 Version:   2.2.103
 Commit:    8edbc2570a

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.13
 OS Platform: Darwin
 RID:         osx.10.13-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.2.103/

Host (useful for support):
  Version: 2.2.1
  Commit:  878dd11e62

.NET Core SDKs installed:
  1.0.1 [/usr/local/share/dotnet/sdk]
  2.2.103 [/usr/local/share/dotnet/sdk]`

**Comment by Varal7** [comment]
I have the same issue, 
- using VS Code Version 1.30.1 (1.30.1)
- with .NET Core SKD version 2.2.104

The launch.json is created but with a placeholder for the "program" entry of "configurations. 

Had to infer from this thread that the correct line should be
```
            "program": "${workspaceFolder}/app/bin/Debug/netcoreapp2.2/app.dll",
```

**Comment by mairaw** [comment]
Thanks for the information everyone! We'll look into this topic more closely as we prepare for .NET Core 3.0 release.

**Comment by tdykstra** [comment]
@BillWagner Your PR #16349 has a `Fixes #9490` comment but this issue didn't get closed automatically.  Should it be closed?

**Comment by BillWagner** [comment]
@tdykstra Yes, this should be closed. I'll close now.
